recipes/qh3/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "qh3" %}
+{% set version = "0.14.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jawah/qh3/archive/v{{ version }}.tar.gz
+  sha256: 871e89b6e6fa8042e2dfbfa9ea21e015f1e4b1c141956183227b02bf0c8b18be
+
+build:
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+    # Adding this here as otherwise it relies on vendor lsqpack
+    - pylsqpack
```

Comment:
This should be in host

Reply:
let me try that

Reply:
The compiler needs to stay in build.

Reply:
got it I'll move it back

Reply:
According to https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Fpylsqpack-0.3.18-py39hf860d4a_0.conda the package doesn't contain the file in question.

Reply:
strange... I guess I'm stuck then

Reply:
The file is in the original ls-qpack C library: https://github.com/search?q=repo%3Alitespeedtech%2Fls-qpack+lsqpack.h&type=code

Reply:
@xhochy does this mean the C library would need to be brought onto conda first?

Reply:
Yes, we will need that first.

Reply:
Depends on what the build tool is. Is it using autotools?

Reply:
I checked upstream, seems like CMake?